packages/sdk/src/modules/rbac/index.ts

```diff
@@ -699,20 +675,22 @@ export class RBAC extends Base {
 
   /**
    * Fetch all groups.
-   * @param option - The option for fetch groups.
+   * @param options - The option for fetch groups.
    * @returns A promise that resolves when the groups is fetched.
    */
 
-  public async fetchGroups(owner: Address): Promise<FetchRoles[]> {
+  public async fetchGroups(options: FetchGroups): Promise<ResponseFetchGroup[]> {
     try {
+      const { owner } = options;
+      if (!owner) throw new Error('Invalid owner address');
       const api = this._getApi();
       const groups = (await api.query?.['peaqRbac']?.['groupStore'](
         owner
       )) as unknown as Entity[];
       if (!groups) {
         throw new Error(`No group is found of owner: ${owner}`);
       }
-      const responseData: FetchRoles[] = groups?.map((item) =>
+      const responseData: ResponseFetchGroup[] = groups?.map((item) =>
```

Comment:
Fixes the object issue we were having before where using {} when calling the function did not work. Changed the parameter to be an object so this syntax can work, as that is how all our other functions are called.